en/patterns/data-visualizations.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Data Visualizations - Design System Pattern
+_description: Data Visualizations Pattern Symbol provides visualization of data or information in various layouts and representations.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Data Visualizations
+
+Use the Data Visualizations Pattern to present information and data in various chart ways, such as Area, Column,Line, Point, Spline chart.
```

Comment:
I'd rather say "various visual ways" or something like that, chart sounds out of place here as it is not an adjective

Reply:
Also mention that the pattern makes it possible to flip though different series types for example by choosing from the drop down.